cmd/docker.go

```diff
@@ -21,16 +24,19 @@ var dockerCmd = &cobra.Command{
 	Long: `This command acts as a helper for docker run.
 You can start with options that go before the image name
 for docker run itself, i.e --env='VAR=VALUE'. Then you must pass
-the image name and the command you want to exucute on that image.`,
+the image name and the command you want to execute on that image.`,
 }
 
-var dockerFlags = &DockerFlags{false, []string{}}
+var dockerFlags = &DockerFlags{false, []string{}, []string{}, "", ""}
 
 func init() {
 	rootCmd.AddCommand(dockerCmd)
 
 	dockerCmd.Flags().BoolVarP(&dockerFlags.DisableTty, "disable-tty", "T", false, "Disables TTY")
 	dockerCmd.Flags().StringArrayVarP(&dockerFlags.EnvVariables, "env", "e", []string{}, "Environment variables")
+	dockerCmd.Flags().StringArrayVarP(&dockerFlags.Volumes, "volume", "v", []string{}, "Bind mount a volume")
+	dockerCmd.Flags().StringVar(&dockerFlags.Expose, "expose", "", "Expose a port or a range of ports")
```

Comment:
I don't remember ever using the `--expose` - just wondering do we really need it?